src/pydata_sphinx_theme/__init__.py

```diff
@@ -1129,6 +1129,11 @@ def copy_logo_images(app: Sphinx, exception=None) -> None:
             continue
         if not (Path(app.srcdir) / path_image).exists():
             logger.warning(f"Path to {kind} image logo does not exist: {path_image}")
+        if path_image.lower().endswith("_t"):
```

Comment:
```suggestion
        # Ensure templates cannot be passed for logo path to avoid security vulnerability
        if path_image.lower().endswith("_t"):
```